tests/src/test/java/alluxio/master/file/CheckConsistencyIntegrationTest.java

```diff
@@ -206,4 +207,34 @@ public void inconsistentFile() throws Exception {
     Assert.assertEquals(expected, mFileSystemMaster
         .checkConsistency(FILE, CheckConsistencyOptions.defaults()));
   }
+
+  /**
+   * Tests the {@link FileSystemMaster#checkConsistency(AlluxioURI, CheckConsistencyOptions)} method
+   * when the UFS file is modified.
+   */
+  @Test
+  public void inconsistentFileModifiedTime() throws Exception {
+    mFileSystem.delete(FILE);
+    FileOutStream fileOutStream = mFileSystem.createFile(FILE, FILE_OPTIONS);
+    fileOutStream.write('a');
+    fileOutStream.close();
+    String ufsFile = mFileSystem.getStatus(FILE).getUfsPath();
+    UnderFileSystem ufs = UnderFileSystem.Factory.create(ufsFile);
+
+    InputStream is = mFileSystem.openFile(FILE);
+    is.read();
+    is.close();
```

Comment:
Beyond i want to let the FILE in the Alluxio cache, so that the checkConsistencyInternal method will enter the branch which i want to test.